contrib/windows/build-installer.iss

```diff
@@ -147,6 +147,12 @@ end;
 
 procedure CurPageChanged(CurPageID: Integer);
 begin
+  ;  Fixing bug in git Issue #48521
```

Comment:
The comments would better describe the behavior of the following code directly.
Something like: change button `next` to `install` when there are no more options.

Reply:
> The comments would better describe the behaviour of the following code directly. Something like: change button text from `next` to `install` when there are no more options.

yes Surely, thank you for the suggestion.
i did changed the comment.

Reply:
> My bad for not expressing myself clearly.
> 
> Delete line 150 ~ line 155. Line 160 can replace them completely.

yes got it!
i hope its ready to merge now.